include/os/freebsd/spl/sys/kmem.h

```diff
@@ -75,15 +75,16 @@ typedef struct kmem_cache {
 extern uint64_t spl_kmem_cache_inuse(kmem_cache_t *cache);
 extern uint64_t spl_kmem_cache_entry_size(kmem_cache_t *cache);
 
-__attribute__((alloc_size(1)))
-void *zfs_kmem_alloc(size_t size, int kmflags);
+void *zfs_kmem_alloc(size_t size, int kmflags)
+    __attribute__((malloc, alloc_size(1)));
 void zfs_kmem_free(void *buf, size_t size);
 uint64_t kmem_size(void);
 kmem_cache_t *kmem_cache_create(const char *name, size_t bufsize, size_t align,
     int (*constructor)(void *, void *, int), void (*destructor)(void *, void *),
     void (*reclaim)(void *) __unused, void *private, vmem_t *vmp, int cflags);
 void kmem_cache_destroy(kmem_cache_t *cache);
-void *kmem_cache_alloc(kmem_cache_t *cache, int flags);
+void *kmem_cache_alloc(kmem_cache_t *cache, int flags)
```

Comment:
It turns out that checkstyle is not happy with putting the attribute on the next line. I will change it to put it on the preceding line in a revision later today.